src/Explorer/Notebook/NotebookContainerClient.ts

```diff
@@ -75,6 +78,12 @@ export class NotebookContainerClient {
             freeKB: memoryUsageInfo.free,
           };
         }
+      } else if (NotebookUtil.isPhoenixEnabled()) {
```

Comment:
>isPhoenixEnabled [](http://example.com/codeflow?start=30&length=16)

why this check here?

Reply:
For old flow we don't have this status update n all. Should happen only in case of phoenix.